src/basics.md

```diff
@@ -1415,10 +1442,14 @@ fn main() {
 [`File::open`]: https://doc.rust-lang.org/std/fs/struct.File.html#method.open
 [`File::try_clone`]: https://doc.rust-lang.org/std/fs/struct.File.html#method.try_clone
 [`File`]: https://doc.rust-lang.org/std/fs/struct.File.html
+[`FixedOffset`]:
```

Comment:
- please keep the reference links in a single line
- I'd suggest to qualify the name with at least one more hierarchy segment